bids_prov/spm_load_config.py

```diff
@@ -38,12 +38,15 @@ def get_empty_graph(context_url=CONTEXT_URL):
         "records": {
             "prov:Agent": [
                 {
-                    "@id": "RRID:SCR_007037",  # TODO query for version
+                    "@id": "exampleAgentID",
```

Comment:
"exampleAgentID" should be replaced by a UUID

Reply:
UUID for SPM ?